bundles/org.openhab.binding.androidtv/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,188 @@
+			<channel id="pincode" typeId="pincode"/>
+			<channel id="app" typeId="app"/>
+			<channel id="appname" typeId="appname"/>
+			<channel id="appurl" typeId="appurl"/>
+			<channel id="player" typeId="player"/>
+			<channel id="power" typeId="system.power"/>
+			<channel id="volume" typeId="system.volume"/>
+			<channel id="mute" typeId="system.mute"/>
+		</channels>
+
+		<properties>
+			<property name="Device Name">unknown</property>
+			<property name="Device ID">unknown</property>
+			<property name="Architectures">unknown</property>
+			<property name="Manufacturer">unknown</property>
+			<property name="Model">unknown</property>
+			<property name="Android Version">unknown</property>
+			<property name="Remote Server">unknown</property>
+			<property name="Remote Server Version">unknown</property>
+		</properties>
+
+		<config-description>
+			<parameter name="ipAddress" type="text" required="true">
+				<context>network-address</context>
+				<label>Hostname</label>
+				<description>Hostname or IP address of the device</description>
+			</parameter>
+			<parameter name="port" type="integer">
+				<label>Port</label>
+				<description>Port to connect to</description>
+			</parameter>
+			<parameter name="keystoreFileName" type="text">
+				<label>Keystore File Name</label>
+				<description>Java keystore containing key and certs</description>
+			</parameter>
+			<parameter name="keystorePassword" type="text">
+				<context>password</context>
+				<label>Keystore Password</label>
+				<description>Password for the keystore file</description>
+			</parameter>
+			<parameter name="reconnect" type="integer">
+				<label>Reconnect Delay</label>
+				<description>Delay between reconnection attempts</description>
+			</parameter>
+			<parameter name="heartbeat" type="integer">
+				<label>Hearbeat Frequency</label>
+				<description>Frequency of heartbeats</description>
+			</parameter>
+			<parameter name="delay" type="integer">
+				<label>Delay</label>
+				<description>Delay between messages</description>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<thing-type id="googletv">
+
+		<label>GoogleTV</label>
+		<description>GoogleTV</description>
+
+		<channels>
+			<channel id="debug" typeId="debug"/>
+			<channel id="keypress" typeId="keypress"/>
+			<channel id="keyboard" typeId="keyboard"/>
+			<channel id="keycode" typeId="keycode"/>
+			<channel id="pincode" typeId="pincode"/>
+			<channel id="app" typeId="app"/>
+			<channel id="player" typeId="player"/>
+			<channel id="power" typeId="system.power"/>
+			<channel id="volume" typeId="system.volume"/>
+			<channel id="mute" typeId="system.mute"/>
+		</channels>
+
+		<properties>
+			<property name="Manufacturer">unknown</property>
+			<property name="Model">unknown</property>
+			<property name="Android Version">unknown</property>
+			<property name="Remote Server">unknown</property>
+			<property name="Remote Server Version">unknown</property>
```

Comment:
Please use camel-case for the property names, see:

https://next.openhab.org/docs/developer/guidelines.html#naming-convention